api/v1/_submissions/PKPBackendSubmissionsHandler.inc.php

```diff
@@ -67,109 +76,129 @@ public function authorize($request, &$args, $roleAssignments)
     }
 
     /**
-     * Get a list of submissions according to passed query parameters
+     * Get a collection of submissions
      *
      * @param $slimRequest Request Slim request object
      * @param $response Response object
+     * @param array $args arguments
      *
      * @return Response
      */
     public function getMany($slimRequest, $response, $args)
     {
-        $request = $this->getRequest();
+        $request = Application::get()->getRequest();
         $currentUser = $request->getUser();
         $context = $request->getContext();
 
-        // Merge query params over default params
-        $defaultParams = [
-            'count' => 20,
-            'offset' => 0,
-        ];
+        if (!$context) {
```

Comment:
Early return ❤️

Reply:
Why not check if `$currentUser` isnt filled too?

Reply:
The `authorization` method, along with the roles defined in the [endpoint](https://github.com/pkp/pkp-lib/blob/main/api/v1/_submissions/PKPBackendSubmissionsHandler.inc.php#L36-L43), should prevent the request from reaching this point unless there is a logged in user.